plutus-core/plutus-ir/test/TransformSpec.hs

```diff
@@ -139,6 +139,13 @@ inline =
     , "single"
     , "immediateVar"
     , "immediateApp"
+    -- these tests are all let bindings of functions, for testing call site inlining of fully
+    -- applied functions.
+    ,"letFunConst" -- const fun fully applied
```

Comment:
Note that I think we do allow comments in PIR files? So you can comment in the input file what you're expecting. I don't think we make much use of that at the moment, but maybe we should.

Reply:
Will do.

Reply:
So I just found out that the parser can't seem to handle comments. I don't know why atm, I checked the code and it looks like it should. :( Should I digress yet again to fix the parser? It seems we want to document the tests a bit more.

Reply:
nvm, I think I found the problem, will push a fix.